packages/integration-sdk-cli/src/__tests__/util/synchronization.ts

```diff
@@ -22,7 +22,7 @@ export function setupSynchronizerApi({ polly, job, baseUrl }: SetupOptions) {
     .post(`${baseUrl}/persister/synchronization/jobs/${job.id}/entities`)
     .intercept((req, res) => {
       allowCrossOrigin(req, res);
-      job.numEntitiesUploaded += JSON.parse(req.body).entities.length;
+      job.numEntitiesUploaded += JSON.parse(req.body!).entities.length;
```

Comment:
The new internal Polly.JS TypeScript definitions allow `body` to be `undefined`. For this test to pass, `body` would need to be defined.